### ISSUE TITLE
How to come back to 4.12 version

### ISSUE THREAD

**Issue by Spok974** [MrChromebox/firmware#283]
Hey i would like to install an hackintosh on cn60 an Cn62 with i7 processor but when I install the last version 4.13 I can't do it ... I ve done It easily on the past with older version ... 
How can I process to install an older version?! 

Thanks for all your awesome work

**Comment by MrChromebox** [comment]
no way to install older versions, keeping track of one version for each of the ~100 device supported is hard enough. Not sure what's different but I know others have installed MacOS without issue on the 4.13 firmware.

**Comment by Spok974** [comment]
Thanks for the answers ! I know a lot of people who have the same issue like me ... if you have a good tutorial or tricks, I could try to make a YouTube video for help the others if I can do it ! Thanks

**Comment by MrChromebox** [comment]
I have no idea, I don't test against MacOS. Check the chrultrabook subreddit or other Hachintoshing forums

**Comment by Spok974** [comment]
Thank you 
Every tutorial I find on the web are under 4.12 version and its work great like my older hackintosh but with 4.13 impossible to do it actually 

It is possible to copy a bios with 4.12 firmware with your online script? If I find somebody who has the same chromebox can I downgrade 4.13 to 4.12 with a copy of the bios?

I hope you anderstand why I mean 😄
Thanks you so much bro

**Comment by MrChromebox** [comment]
no, my script doesn't provide option to flash older versions, and if you use the firmware from another device you will run into issues with the serial number and LAN MAC address being copied

**Comment by one8three** [comment]
> I know others have installed MacOS without issue on the 4.13 firmware

@MrChromebox, do you happen to have a source you can point me to for this? Everything I've found points to the same issue with 4.13 and MacOS (kernel panics on reboots/startup and during installation). My best guess is still that it's NVRAM related but doesn't seem to be specific to a model. Maybe generation specific though (Broadwell).

**Comment by MrChromebox** [comment]
I don't, I don't use MacOS. I have no way of testing 100+ devices (or even 10+ platforms) for each release.

**Comment by one8three** [comment]
Yeah, I get that. Was just hoping you could point me in the direction of somebody who has had success with MacOS and 4.13 so I could compare notes.

**Comment by toanlm1981** [comment]
@TheRandMan did you try 4.14 version?

**Comment by Spok974** [comment]
Waldoxhm said: bigsur version of efi is updated. I've fully tested with coreboot 4.14, I got kernel panic when boot up, it may due to the bugfix of IGPU and the way nvram stores changed. I managed to downgrade bios to 4.12 check https://github.com/waldoxhm/scripts. 1.burn a linux image to a sdcard with balenaEtcher. 2.boot cn62 to this livecd os. 3.follow the guide run the command.

**Comment by MrChromebox** [comment]
@Spok974 that repo is a bad idea, there's no guarantee I'll keep those older versions around. It also won't maintain the Ethernet MAC address on Chromeboxes since the script logic isn't set up to handle going back to the older VPD format

**Comment by waldoxhm** [comment]
@TheRandMan simplely set <code>Booter->Quirks->DisableVariableWrite</code> and <code>NVRAM->LegacyEnable</code> to <code>true</code> in config.plist will do the trick in my case. And use emulated NVRAM with LogoutHook.

**Comment by one8three** [comment]
> @TheRandMan simplely set <code>Booter->Quirks->DisableVariableWrite</code> and <code>NVRAM->LegacyEnable</code> to <code>true</code> in config.plist will do the trick in my case. And use emulated NVRAM with LogoutHook. 

Thought I tried this but will try again. Thanks.
